### PR TITLE
UNI-319: Windows isolated build

### PR DESCRIPTION
**Pull request by lscheinkman**
@oxtopus Please review. 
Here is the bamboo build https://ci.numenta.com/browse/UN-UPT0-14

CC: @marionleborgne

### CODE REVIEW

File: unicorn/README.md [hunk 1]
```diff
@@ -410,14 +407,138 @@ The resulting artifacts can be found in `dist/`
...
+net use x: \\VBOXSVR\shared /PERSISTENT:YES
+```
+
+This shared folder is using [VirtualBox shared folder](https://www.vagrantup.com/docs/synced-folders/virtualbox.html) feature
+which syncs file changes from the guest to the host and vice versa.
+
+
+##### Building the application
+
+Vagrant uses uses `winrm` to communicate with Windows virtual machines therefore
```
Comment (oxtopus) [inline unicorn/README.md]: Typo: "uses uses"

File: unicorn/windows/build_python.ps1 [hunk 2]
```diff
@@ -22,39 +22,19 @@
...
-
 Write-Host "Mount shared folder to 'x:' drive"
 # Must match shared folder name in vagrant file
 # See 'config.vm.synced_folder')
 net use x: \\VBOXSVR\shared /PERSISTENT:YES
-pushd x:\
 
 Write-Host "Build python and nupic"
-pushd scripts\Windows64
+pushd x:\scripts\Windows64
```
Comment (oxtopus) [inline unicorn/windows/build_python.ps1]: Perhaps include a comment or other documentation that makes it clear that this script either _only_ works in the vagrant-managed VM, or that there are special drive mappings and other steps that a user must take if they were to run this in their own windows environment.

**Comment by oxtopus** [comment]
Only minor documentation-related issues, otherwise LGTM 👍

File: unicorn/README.md [hunk 3]
```diff
@@ -410,14 +407,145 @@ The resulting artifacts can be found in `dist/`
...
+```shell
+vagrant reload
+```
+
+Once the VM is up and running, the application source code be available from
+the `\\VBOXSVR\shared` shared folder which should be mounted to `x:` drive.
+
+You can remount the shared folder using the following command:
+```
+net use x: \\VBOXSVR\shared /PERSISTENT:YES
```
Comment (marionleborgne) [inline unicorn/README.md]: @lscheinkman shouldn't this be: `vagrant winrm -c 'net use x: \\VBOXSVR\shared /PERSISTENT:YES'` ?

Comment (lscheinkman) [inline unicorn/README.md]: `winrm` is just the remote shell version for windows.
 I'm just pointing out you can remap the shared folder, how you run the command is up to you. Either by login to the machine via `rdp` or remote shell via `winrm`

**Comment by lscheinkman** [comment]
@marionleborgne Were you able to build on your laptop?

**Comment by marionleborgne** [comment]
@lscheinkman After trying to `vagrant up` on a fresh environment, I noticed that `ModernIE/w10-edge` has been revoked. https://atlas.hashicorp.com/modernIE/boxes/w10-edge

See post: https://joecod.es/retiring-modernie-vagrant-boxes/

> I've made the decision to officially retire any future updates and support for the ModernIE Vagrant boxes I have uploaded at https://atlas.hashicorp.com/ModernIE

**Comment by oxtopus** [comment]
This PR has been open for a while, perhaps hinging on whether or not @marionleborgne was able to use it.

@lscheinkman @marionleborgne can you guys check in on this one?

**Comment by rhyolight** [comment]
This PR will become much more important as soon as we turn off AppVeyor @scottpurdy

**Comment by vitaly-krugl** [comment]
There seems to be some disconnect between JIRA UNI-319, which is in "closed" state, and this PR, which hasn't been accepted/merged.

**Comment by lscheinkman** [comment]
Isolation will be done on our current CI (Bamboo). This PR is obsolete

**Pull request closed without merging**